dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/dbt-labs/dbt-adapters.git@main
+git+https://github.com/dbt-labs/dbt-adapters.git@refactor-magic-catalog-int
```

Comment:
can be restored once https://github.com/dbt-labs/dbt-adapters/pull/131 is merged.